modules/oadp-usecase-kopia-override-algorithms.adoc

```diff
@@ -0,0 +1,178 @@
+// Module included in the following assemblies:
+//
+// * backup_and_restore/application_backup_and_restore/installing/overriding-kopia-algorithms.adoc          
+
+:_mod-docs-content-type: PROCEDURE
+[id="oadp-usecase-kopia-override-algorithms.adoc_{context}"]
+= Use case for overriding Kopia hashing, encryption, and splitter algorithms
+
+This use case walks you through the steps for taking a backup with Kopia environment variables for hashing, encryption, and splitter. You store the backup in an AWS S3 bucket. You then verify the environment variables by connecting to the Kopia repository.
```

Comment:
"This might be a vague referent" and "walks" might be confusing to "non-native-English" speakers. How about:

> The use case example demonstrates making a backup <of an application?> by using Kopia environment variables for hashing, encryption, and splitter tasks.

Reply:
Done.